uni/assets/text/TermsAndConditions.md

```diff
@@ -4,50 +4,144 @@
+To the fullest extent permitted by applicable law, in no event will NIAEFEUP, its affiliates,
+officers, directors, employees, agents, suppliers or licensors be liable to any person for (a): any
+indirect, incidental, special, punitive, cover or consequential damages (including, without
+limitation, damages for lost profits, revenue, sales, goodwill, use of content, impact on business,
+business interruption, loss of anticipated savings, loss of business opportunity) however caused,
+under any theory of liability, including, without limitation, contract, tort, warranty, breach of
+statutory duty, negligence or otherwise, even if NIAEFEUP has been advised as to the possibility of
+such damages or could have foreseen such damages. To the maximum extent permitted by applicable law,
+the aggregate liability of NIAEFEUP and its affiliates, officers, employees, agents, suppliers and
+licensors, relating to the services will be limited to an amount greater of one dollar or any
+amounts actually paid in cash by you to NIAEFEUP for the prior one month period prior to the first
+event or occurrence giving rise to such liability. The limitations and exclusions also apply if this
+remedy does not fully compensate you for any losses or fails of its essential purpose.
 \
 \
 **Changes and amendments**
 
-We reserve the right to modify this Agreement or its policies relating to the Mobile Application or Services at any time, effective upon posting of an updated version of this Agreement in the Mobile Application. When we do, we will revise the updated date at the bottom of this page. Continued use of the Mobile Application after any such changes shall constitute your consent to such changes. Policy was created with [Website Policies](https://www.websitepolicies.com).
+We reserve the right to modify this Agreement or its policies relating to the Mobile Application or
+Services at any time, effective upon posting of an updated version of this Agreement in the Mobile
+Application. When we do, we will revise the updated date at the bottom of this page. Continued use
+of the Mobile Application after any such changes shall constitute your consent to such changes.
+Policy was created with [Website Policies](https://www.websitepolicies.com).
 \
```

Comment:
Here we can make it more clear that the user will be prompted to accept the revised terms and has the option not to accept them and logout.